modules/ebi-metagenomics/keggpathwayscompleteness/main.nf

```diff
@@ -0,0 +1,77 @@
+
+process KEGGPATHWAYSCOMPLETENESS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kegg-pathways-completeness:1.0.4--pyhdfd78af_0':
+        'biocontainers/kegg-pathways-completeness:1.0.4--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    tuple val(meta), path(hmmsearch_tbl)
+    tuple val(meta), path(ko_list)
```

Comment:
I think if the 3 inputs need the meta, maybe there should only be one input: tuple val(meta), path(fasta), path(hmmsearch_tbl), path(ko_list) as otherwise using this to process "many" samples won't work as it won't be possible to process the same sample with 3 different inputs.